components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt.ui/src/main/resources/org/wso2/carbon/identity/workflow/mgt/ui/i18n/Resources.properties

```diff
@@ -132,6 +132,7 @@ workflow.error.not.select.operation.category=Operation Category must be select
 workflow.error.not.select.operation.name=Operation Name must be select
 workflow.error.not.select.workflow=Workflow must be select
 workflow.error.empty.workflow.name=Workflow Name can not be empty
+workflow.error.duplicate.workflow.name=Workflow Name can not be duplicate
```

Comment:
Workflow Name can not be duplicated or Workflow Name can not be duplicate a duplicate name. 
Grammar is wrong

Reply:
Fixed